pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.en-gb.md

```diff
@@ -98,13 +137,29 @@ $ openstack loadbalancer listener create --name stats-listener --protocol PROMET
 +-----------------------------+--------------------------------------+
 ```
 
+Note that you add the `--allowed-cidr` option in order to filter the listener to a specific network.
```

Comment:
Maybe we should highlight that the metric listen will be exposed on Internet (if a FIP is attached) ?